jvm/controls/src/main/kotlin/org/wycliffeassociates/otter/jvm/controls/styles/AppStyles.kt

```diff
@@ -30,6 +30,7 @@ class AppStyles : Stylesheet() {
         fun bookIcon(size: String = "1em") = MaterialIconView(MaterialIcon.BOOK, size)
         fun chapterIcon(size: String = "1em") = MaterialIconView(MaterialIcon.CHROME_READER_MODE, size)
         fun verseIcon(size: String = "1em") = MaterialIconView(MaterialIcon.BOOKMARK, size)
+        fun closeIcon(size: String = "1em") = MaterialIconView(MaterialIcon.CLOSE, size)
```

Comment:
I'd really prefer we try to slowly move everything out of this file and use CSS, then delete this file when nothing uses it anymore. So I definitely don't want more things to use it.